lib/adapters/idb/idb-utils.js

```diff
@@ -63,7 +63,7 @@ exports.decodeMetadata = function (storedObject) {
   }
   var metadata = utils.safeJsonParse(storedObject.data);
   metadata.winningRev = storedObject.winningRev;
-  metadata.deletedOrLocal = storedObject.deletedOrLocal === '1';
+  metadata.deleted = storedObject.deletedOrLocal === '1';
```

Comment:
I was tired of the metadata exposing this internal implementation detail of the IDB adapter: that our "deleted" index is named "deletedOrLocal" for backwards-compatibility reasons. Prefer not to expose this weird name anymore.